coorx/base_transform.py

```diff
@@ -116,7 +121,7 @@ def systems(self):
         return self._systems
 
     def set_systems(self, from_cs, to_cs, cs_graph=None):
-        assert (from_cs is None) == (to_cs is None), "from_cs and to_cs must both be None or both be str"
+        assert (from_cs is None) == (to_cs is None), "from_cs and to_cs must both be None or both be coordinate systems"
         if from_cs is not None:
```

Comment:
What happens if we call set_systems more than once on a transform? Especially with None after a system has already been set? Maybe we just disallow this for now (can call again, but only if the args match the existing values)

Reply:
Oh, this is already checked in CoordinateSystemGraph